skbio/io/format/genbank.py

```diff
@@ -45,6 +45,8 @@
 section, its value is stored as a list, as there are often multiple
 reference sections in one GenBank record.
 
+.. _genbank_feature_section:
+
```

Comment:
Sorry, I couldn't comment this piece of code in sphinx documentation. This is a [inline markup](http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-arbitrary-locations) to refer to an arbitrary location, and let me to reference to a precise point of another document in text by using
```
See :ref:`Genbank FEATURES section<genbank_feature_section>`
```
(note the absence of the initial `_` in the link). The markup need to be placed immediately before the title I need to refer to. My idea was, since features were read in the same way as genbank, to refer to genbank documentation to explain `FT section`. Feel free to ask me to modify this aspect and describe `FT` in embl module